Solutions/ESETPROTECT/Analytic Rules/ESETThreatDetected.yaml

```diff
@@ -0,0 +1,40 @@
+id: 64badfab-1dd8-4491-927b-3ca206fa9a17
+name: Threats detected by ESET
+description: |
+  'Escalates threats detected by ESET.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: ESETPROTECT
+    dataTypes:
+      - Syslog
+queryFrequency: 5m
+queryPeriod: 5m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Execution
+  - CredentialAccess
+  - PrivilegeEscalation
+query: |
+  ESETPROTECT
+  | where EventType == "Threat_Event"
+  | extend HostCustomEntity = DvcHostname, AccountCustomEntity = SrcUserName, IPCustomEntity = DvcIpAddr, FileHashCustomEntity = FileHashSha1, FileHashAlgo = "SHA1"
+entityMappings:
+  - entityType: Account
+    fieldMappings:
+      - identifier: FullName
+        columnName: AccountCustomEntity
+  - entityType: Host
+    fieldMappings:
+      - identifier: FullName
+        columnName: HostCustomEntity
+  - entityType: IP
+    fieldMappings:
+      - identifier: Address
+        columnName: IPCustomEntity
+  - entityType: FileHash
+    fieldMappings:
+      - identifier: Algorithm
+        columnName: FileHashAlgo
+      - identifier: Value
+        columnName: FileHashCustomEntity
```

Comment:
@dmaasland, Please add field **Version** in Analytic rule file as mentioned below
"version: 1.0.0"
refer this link for reference. [Sample PR link](https://github.com/Azure/Azure-Sentinel/pull/2271)